src/store/global.ts

```diff
@@ -15,9 +15,12 @@ export const global = createSlice({
     setIsLoggedIn: (state, { payload: { isLoggedIn } }) => {
       state.isLoggedIn = isLoggedIn;
     },
+    setIsLoggedOut: (state, { payload: { isLoggedIn } }) => {
+      state.isLoggedIn = isLoggedIn;
+    }
```

Comment:
why a different action for this, can we not use the above one and pass false

Reply:
yes we can i will remove it